python/sunbeam/schedule.py

```diff
@@ -131,6 +131,11 @@ def wells(self):
         names = self._wellnames(self.timestep)
         return [w for w in self._schedule.wells if w.name in names]
 
+    @property
+    def vfp_table(self):
```

Comment:
Since this is not actually *a table* - but rather a table index / table id or someting; maybe a name like `vfp_table_nr` would be better?